omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentController1_10Test.java

```diff
@@ -257,4 +259,35 @@ public void postAttachment_shouldNotUploadFileAboveSizeLimit() throws Exception
 		// Replay
 		SimpleObject response = deserialize(handle(request));
 	}
+	
+	@Test
+	public void getAttachmentBytes_shouldDownloadFile() throws Exception {
+		
+		// Setup
+		String fileCaption = "Test file caption";
+		String uuid;
+		
+		{// Upload Test File
```

Comment:
The isolating scope could be removed here. Since there is only one, its code could as well be part of the main scope.